internal-payment-sdk/sdk/src/test/java/net/gini/android/internal/payment/paymentComponent/PaymentComponentViewTest.kt

```diff
@@ -68,38 +68,6 @@ class PaymentComponentViewTest {
         scenario!!.close()
     }
 
-    @Test
-    fun `calls onMoreInformation method of listener when clicking on info button`() = runTest {
```

Comment:
[Clarification] Do we not need these tests anymore? Or are they being moved to another class?